resource/user/user.go

```diff
@@ -217,19 +183,10 @@ func (u *User) Check(context.Context, resource.Renderer) (resource.TaskStatus, e
 
 // Apply changes for user
 func (u *User) Apply(context.Context) (resource.TaskStatus, error) {
```

Comment:
why couldn't apply use the status that you've created already, again? I feel like we had this discussion, but a) I don't remember and b) has the answer changed?